### ISSUE TITLE
Inconsistent behaviour with IMAP rename: Duplicate Messages

### ISSUE THREAD

**Issue by sralpert** [cleidigh/EditEmailSubject-MX#56]
My connection uses IMAP in and SMTP out!

I sent a simple file to myself (in my Tbird inbox):
![Inbox - Steve - Mozilla Thunderbird-20220211_110429](https://user-images.githubusercontent.com/6002578/153628595-6fd07945-83b0-4c1a-a179-0828f8d7d5ed.png)

Checking Spectrum webmail:
![sralpert@charter net - Webmail - Personal - Microsoft​ Edge-20220211_110443](https://user-images.githubusercontent.com/6002578/153628616-e408055a-2014-4a10-b915-c67824c10aaf.png)

Used plugin to edit title:
![Inbox - Steve - Mozilla Thunderbird-20220211_110529](https://user-images.githubusercontent.com/6002578/153628650-7a6eb097-dfd5-4d1c-ab76-a3e6715a55ba.png)

See that original is not shown, but looking at webmail:
![sralpert@charter net - Webmail - Personal - Microsoft​ Edge-20220211_110540](https://user-images.githubusercontent.com/6002578/153628694-f9465fd2-2192-4e8c-b22f-cf3bddb4e4cf.png)

I then moved Test message with edited subject in Tbird to a local folder with drag & drop and refreshed web email:
Nothing showed in Tbird inbox but webmail showed BOTH
![sralpert@charter net - Webmail - Personal - Microsoft​ Edge-20220211_110720](https://user-images.githubusercontent.com/6002578/153628726-00722ccc-a7f1-44b5-babe-3f2b3a1f436a.png)

I then STOPPED and restarted thunderbird. At that point, the messages disappeared from webmail
and from Tbird inbox. The messages don't appear in the webmail trash.
Were they "erased" when I stopped Tbird?

**Comment by bege10** [comment]
Today it is the first time that I have a similar behavior:

- The original message is still there, additionally a copy with the changed subject.
- In another account the message completely disappeared after clicking OK.

Neither  repairing the folders nor restarting TB helped.
For the moment I deactivate the extension. 😢

Windows 10 Professional 64 bit
TB 102.12.0

**Comment by jobisoft** [comment]
I added a backup option (enabled by default) to allow users to get back the original message. I will keep investigating the actual issue reported here.

https://github.com/cleidigh/EditEmailSubject-MX/commit/c50d42d721e7178afd1c4b4f997c35af69c18162

**Comment by jobisoft** [comment]
Could you check, if this still happens for your with version 6.2?
https://github.com/cleidigh/EditEmailSubject-MX/releases/tag/v6.2

**Comment by alelaf** [comment]
I have the exact same issue since few days, email doubling when editing subject, and one have the old subject, the other one have the new subject.
I used this extension for a while and never got issues previously.

Thanks

**Comment by jobisoft** [comment]
Please try v7.0:
https://github.com/cleidigh/EditEmailSubject-MX/releases/tag/v7.0

Does that fix your issue?

**Comment by mbrouillet** [comment]
I have the same issue : old message remains, in addition to existing.
I see that my setting of backup in ESS-Temp is active. I guessed maybe the copy remains in main folder if ESS-Tepm does not exist. I didn't see any such folder.
- I did create this folder under Inbox ; still double message.
- I placed the ESS-Temp in the top level of the mailbox ; still double message.
TB 102.15.1 on Ubuntu ; EES 6.2

I guess I have to be patient until TB 115 gets released under Ubuntu 22.04 ?
